__tests__/app.test.js

```diff
@@ -145,12 +145,47 @@ describe("GET /api/articles/articles_id/comments", () => {
         expect(allComments).toBeSortedBy("created_at", { descending: true });
       });
   });
-  test("GET 404: should give an appropriate error message when given an invalid endpoint", () => {
+  test("404: should give an appropriate error message when given an invalid endpoint", () => {
     return request(app)
       .get("/api/articles/1/banana")
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe("Incorrect endpoint")
+        expect(body.msg).toBe("Incorrect endpoint");
+      });
+  });
+});
+
+describe.only("POST /api/articles/article_id/comments", () => {
```

Comment:
Please remember to remove all the `.only` from the test suite! As it's on the describe block I will go through this but we may ask for a re-submission in the future